chpl/chpl-api/e2e/collections/products-controller.postman_collection.json

```diff
@@ -89,6 +138,254 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "product schema and response validation_GET_/products",
```

Comment:
Request name should only have request method and rest end point as mentioned above